src/core/dts.ts

```diff
@@ -6,7 +6,7 @@ export function generateDeclaration(imports: ImportsFlatMap, resolvedImports: Im
     ...Object.entries(resolvedImports),
   ]
     .sort((a, b) => a[0].localeCompare(b[0]))
-    .map(([name, info]) => `  const ${name}: typeof import('${info.from}')${info.name === '*' || !info.name ? '' : `['${info.name || name}']`}`)
+    .map(([name, info]) => `  const ${name}: typeof import('${info.from}')${info.name === '*' || info.name === 'default' ? '' : `['${info.name || name}']`}`)
```

Comment:
There the `['default']` is required since `import()` returns the module

Reply:
some packages may not export default, and the generated auto-import.d.ts will throw an error of ts2339 indicating that default does not exist. In this case, I think it is possible to dynamically add default to import() as needed:
`const dayjs: typeof import('dayjs') `instead of `const dayjs: typeof import('dayjs')['default']`

Reply:
> There the `['default']` is required since `import()` returns the module

some packages may not export default, and the generated auto-import.d.ts will throw an error of ts2339 indicating that default does not exist. In this case, I think it is possible to dynamically add default to import() as needed:
const dayjs: typeof import('dayjs') instead of const dayjs: typeof import('dayjs')['default']

Reply:
Some packages may not export default, and the generated auto-import.d.ts will throw an error of ts2339 indicating that default does not exist. In this case, I think it is possible to dynamically add default to import() as needed:
const dayjs: typeof import('dayjs') instead of const dayjs: typeof import('dayjs')['default']

Reply:
some packages may not export default, and the generated auto-import.d.ts will throw an error of ts2339 indicating that default does not exist. In this case, I think it is possible to dynamically add default to import() as needed:
`const dayjs: typeof import('dayjs') `instead of `const dayjs: typeof import('dayjs')['default']`